core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -200,7 +200,7 @@ private[spark] class TaskSchedulerImpl(
         val execId = taskIdToExecutorId(tid)
         backend.killTask(tid, execId, interruptThread)
       }
-      tsm.abort("Stage %s cancelled".format(stageId))
+      tsm.abort("Stage %d was cancelled".format(stageId))
```

Comment:
Although I'd like to standardize too, I think it's not worth it as I've come to appreciate how annoying merge conflicts are. This just doesn't buy anything.